Blog/style/style.css

```
@@ -54,4 +55,43 @@ header{
 .search-input::placeholder{
   color: var(--dark-50);
   font-family: 'Inter';
-}
\ No newline at end of file
+}
+
+/******************* Seção Cards *****************/
+li{
+  background-color: var(--dark-20);
+  padding: 40px;
+  border: 2px;
+  border-radius: 8px;
+}
+
+ion-icon{
+font-size: 20px;
+}
+
+.data_logo{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: var(--brand-color);
+  font-family: 'Inter';
+  font-size: 16px;
+  font-weight: 500;
+}
+
+#titlle-card{
+  color: var(--dark-60);
+  font-size: 24px;
+  font-weight: 500;
+  font-family: 'Space Grotesk';
+  margin-top: 20px;
+}
+
+#text-card{
+  color: var(--dark-50);
+  font-family: 'Inter';
```

Sempre lembrar de colocar a font "padrão", caso não tenha inter
```suggestion
  font-family: "Inter", "sans-serif";
```